api/build.gradle

```diff
@@ -402,36 +464,36 @@ dependencies {
   compile 'mysql:mysql-connector-java:8.0.16'
   compile 'com.google.cloud.sql:mysql-socket-factory:1.0.10'
 
-  compile "com.google.appengine:appengine:${GAE_VERSION}"
-  compile "com.google.appengine:appengine-api-1.0-sdk:${GAE_VERSION}"
+  compile "com.google.appengine:appengine:$project.ext.GAE_VERSION"
+  compile "com.google.appengine:appengine-api-1.0-sdk:$project.ext.GAE_VERSION"
 
-  compile("org.springframework.boot:spring-boot-starter-web:${SPRING_BOOT_VERSION}") {
+  compile("org.springframework.boot:spring-boot-starter-web:$project.ext.SPRING_BOOT_VERSION") {
     exclude module: 'spring-boot-starter-tomcat'
     exclude group: 'org.slf4j', module: 'jul-to-slf4j'
   }
-  compile("org.springframework.boot:spring-boot-starter-data-jpa:${SPRING_BOOT_VERSION}") {
+  compile("org.springframework.boot:spring-boot-starter-data-jpa:$project.ext.SPRING_BOOT_VERSION") {
     exclude module: 'spring-boot-starter-tomcat'
     exclude group: 'org.slf4j', module: 'jul-to-slf4j'
   }
-  compile("org.springframework.boot:spring-boot-starter-actuator:${SPRING_BOOT_VERSION}") {
+  compile("org.springframework.boot:spring-boot-starter-actuator:$project.ext.SPRING_BOOT_VERSION") {
     exclude module: 'spring-boot-starter-tomcat'
     exclude group: 'org.slf4j', module: 'jul-to-slf4j'
   }
 
   compile 'ch.qos.logback:logback-classic:1.2.3'
 
   compile 'org.springframework.security:spring-security-web:4.2.13.RELEASE'
-  compile "org.hibernate:hibernate-core:${HIBERNATE_VERSION}"
+  compile "org.hibernate:hibernate-core:$project.ext.HIBERNATE_VERSION"
 
   // Force the Kotlin version, otherwise the Kotlin plugin conflicts with Jackson's
   // Kotlin dep, resulting in massive logspam.
-  compile "org.jetbrains.kotlin:kotlin-reflect:${KOTLIN_VERSION}"
+  compile "org.jetbrains.kotlin:kotlin-reflect:$project.ext.KOTLIN_VERSION"
```

Comment:
Why the change to omit curly brackets? I'm surprised this is legal

Reply:
IntelliJ suggests that they're redundant and asks to elide them for me.
![Screen Shot 2020-04-16 at 9 26 01 AM](https://user-images.githubusercontent.com/53479492/79461581-5a171080-7fc4-11ea-9b34-fa4f52012df0.png)

From the [docs](https://groovy-lang.org/syntax.html#_string_interpolation)

> In addition to ${} placeholders, we can also use a lone $ sign prefixing a dotted expression:

def person = [name: 'Guillaume', age: 36]
assert "$person.name is $person.age years old" == 'Guillaume is 36 years old'
But only dotted expressions of the form a.b, a.b.c, etc, are valid. Expressions containing parentheses like method calls, curly braces for closures, dots which aren’t part of a property expression or arithmetic operators would be invalid.